site/docs/es/plugins/runner.md

```diff
@@ -156,8 +156,10 @@ run(bot, { runner: { fetch: { allowed_updates: [] } } });
 
 ## Multihilo
 
-JavaScript es monohilo.
+> No tiene sentido el multithreading si tu bot no procesa al menos 50 millones de actualizaciones al día (>500 por segundo).
+> [Sáltate esta sección](#cómo-funciona-entre-bastidores) si tu bot maneja menos tráfico que eso.
```

Comment:
@habemuscode Broken anchor. It should be `#como-funciona-entre-bastidores`

Reply:
#734